src/Button/GeoLocationButton/GeoLocationButton.jsx

```diff
@@ -0,0 +1,265 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import OlMap from 'ol/map';
+import OlGeolocation from 'ol/geolocation';
+import OlGeomLineString from 'ol/geom/linestring';
+import OlOverlay from 'ol/overlay';
+
+import ToggleButton from '../ToggleButton/ToggleButton.jsx';
```

Comment:
You may want to import the `ToggleButton` from the `index.js`.

Reply:
Done.